openhcl/hcl/src/ioctl/aarch64.rs

```diff
@@ -57,6 +58,7 @@ impl super::BackingPrivate for MshvArm64 {
         runner: &mut ProcessorRunner<'_, Self>,
         name: HvRegisterName,
         value: HvRegisterValue,
+        _vtl: GuestVtl,
```

Comment:
Should we assert the vtl is vtl 0?

Reply:
Except for x18, these registers are all shared across VTLs, so the VTL doesn't matter. I'll have to think carefully about how to handle x18 here. (Its handling here was added as part of ARM emulation bringup; we'll need a VTL-aware place to store it.)

Reply:
Added a `todo!` for right now. I think this will have to be handled as part of a bigger change to make AccessCpuState VTL-aware.